# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/hotfix-system-highway.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-141.0.7390.54/build/linux/unbundle/highway.gn.vanilla	2025-10-04 11:49:26.427065951 +0200
+++ chromium-141.0.7390.54/build/linux/unbundle/highway.gn	2025-10-04 11:50:13.567134846 +0200
@@ -18,3 +18,6 @@
   public_configs = [ ":libhwy_external_config" ]
   public_deps = [ ":libhwy_shim" ]
 }
+
+source_set("highway_tests") {
+}
